samples/condor/Makefile

```diff
@@ -29,6 +51,6 @@ distclean: clean
 distclean-recursive: clean
 
 boinc_gahp: boinc_gahp.cpp $(BOINC_SOURCE_LIB_DIR)/remote_submit.h $(BOINC_SOURCE_LIB_DIR)/remote_submit.cpp $(BOINC_DIR)/svn_version.h $(BOINC_DIR)/version.h
-	$(CXX) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) $(MINGW_FLAGS) -g -O2 -I$(BOINC_DIR) -I$(BOINC_SOURCE_LIB_DIR) \
-	-I$(VCPKG_DIR)/include -o boinc_gahp boinc_gahp.cpp $(BOINC_SOURCE_LIB_DIR)/remote_submit.cpp \
-	-L$(BOINC_LIB_DIR) -L$(VCPKG_DIR)/lib -lcurl $(MINGW_CURL) -lssl -lcrypto -lboinc $(MAKEFILE_LDFLAGS) $(STDCPPTC)
+	$(CXX) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) $(MINGW_FLAGS) -g -O2 \
+	-o boinc_gahp boinc_gahp.cpp $(BOINC_SOURCE_LIB_DIR)/remote_submit.cpp \
+	-lcurl -lssl -lcrypto -lboinc $(CURL_EXTRA_LDFLAGS) $(MAKEFILE_LDFLAGS) $(STDCPPTC)
```

Comment:
In case on non-vcpkg build, '-lcurl', '-lssl' and '-lcrypto' are already specified in 'CURL_EXTRA_LDFLAGS'.
Maybe it's reasonable to remove duplicates?

Reply:
It not specify in the `CURL_EXTRA_LDFLAGS`. If I will remove them from here, I will need to create another variable for all that it will be the same.